test/data/test_error_expected.json

```
@@ -0,0 +1,90 @@
+    "logger": "odoo",
+    "message": "database: default@default:default",
+    "error": false
+  },
+  {
+    "asctime": "2017-05-21 14:01:50,496",
+    "pid": "8038",
+    "levelname": "INFO",
+    "dbname": "?",
+    "logger": "odoo.modules.loading",
+    "message": "init db",
+    "error": false
+  },
+  {
+    "asctime": "2017-05-21 14:01:50,637",
+    "pid": "8038",
+    "levelname": "ERROR",
+    "dbname": "?",
+    "logger": "odoo.modules.registry",
+    "message": "Failed to load registry\nTraceback (most recent call last):\n...\nOSError: [Errno 2] No such file or directory: '/home/nobody/odoo-10.0/addons'",
+    "error": true,
+    "error_ignored": false
+  },
+  {
+    "asctime": "2017-05-21 14:01:50,637",
+    "pid": "8038",
+    "levelname": "CRITICAL",
+    "dbname": "?",
+    "logger": "odoo.service.server",
+    "message": "Failed to initialize database `odoo-bscw-testsuite-1`.\nTraceback (most recent call last):\n...",
+    "error": true,
+    "error_ignored": false
+  },
+  {
+    "asctime": "2017-05-21 14:01:50,639",
+    "pid": "8038",
+    "levelname": "WARNING",
+    "dbname": "?",
+    "logger": "odoo.addons.base.models.res_currency",
+    "message": "The num2words python library is not installed, amount-to-text features won't be fully available.\nException in thread odoo.service.httpd:\nTraceback (most recent call last):\nFile \"/usr/lib/python3.7/threading.py\", line 926, in _bootstrap_inner\nself.run()\n...",
+    "error": true,
+    "error_ignored": false
+  },
+  {
+    "asctime": "2017-05-21 14:01:50,638",
+    "pid": "8038",
+    "levelname": "INFO",
+    "dbname": "?",
+    "logger": "odoo.service.server",
+    "message": "Initiating shutdown",
+    "error": false
+  },
+  {
+    "asctime": "2017-05-21 14:01:50,638",
+    "pid": "8038",
+    "levelname": "INFO",
+    "dbname": "?",
+    "logger": "odoo.service.server",
+    "message": "Hit CTRL-C again or send a second signal to force the shutdown.",
+    "error": false
+  }
+]
```

A test showing ignored errors would be useful.